pilot/pkg/xds/ads.go

```diff
@@ -992,6 +987,24 @@ func (conn *Connection) Watched(typeUrl string) *model.WatchedResource {
 	return nil
 }
 
+// eventTypes return the event types that need to pushed when config changes.
+func (conn *Connection) eventTypes() []EventType {
```

Comment:
The function does not return expected all types, so looks confusing. Maybe `AllEventTypes` is not right. @howardjohn

Reply:
Sorry I did not understand. This is to support this call https://github.com/istio/istio/blob/master/pilot/pkg/xds/ads.go#L694 where we need to push all eventTypes.